cnames_active.js

```diff
@@ -729,6 +729,7 @@ var cnames_active = {
   "discordjs-handler": "discordjs-handler.github.io/website",
   "discordlib": "hosting.gitbook.com",
   "discords": "botstudios.github.io/discords.js.org",
+  "discuss": "cname.vercel-dns.com",
```

Comment:
```suggestion
  "discuss": "cname.vercel-dns.com", // noCF
```
Based on other instances here, I suspect you want the `// noCF` flag

Reply:
@MattIPv4 Thats not a must as it works also with the proxy: https://discuss.js.org
Lets see what @Lete114 has to say...

Reply:
Oh interesting, I wonder if something changed on Vercel's end then -- might be related to that issue about the warning emails I wonder

Reply:
I don't need the `// noCF` flag

Reply:
I went to bed because of the time zone, and I'm sorry to have kept you waiting so long

Reply:
> @MattIPv4  Oh interesting, I wonder if something changed on Vercel's end then -- might be related to that issue about the warning emails I wonder

I don't know, I get a warning email when I use my own domain